installation.rst

```diff
@@ -296,3 +288,29 @@ After setting the node up, to bring it back you need to run a Juju *action*:
 
 Please refer to our :ref:`operations` section for detailed instructions on how
 to manage the cluster.
+
+Local charms
```

Comment:
want to move this to the development docs?

Reply:
I will do that in the next PR. This one here is not aware of that document yet.